spec/datetime.html

```diff
@@ -1,352 +1,824 @@
+        1. Let _timeZone_ be ? ToTimeZone(_timeZoneParam_).
+        1. If _disambiguation_ is not *undefined*, then
+          1. Set _disambiguation_ to ? ToString(_disambiguation_).
+        1. Else
+          1. Set _disambiguation_ to `"earlier"`.
+        1. Return ? GetAbsoluteFor(_timeZone_, _dateTime_, _disambiguation_).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause>
+      <h1>Temporal.DateTime.prototype.getDate ( )</h1>
+      <p>
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _dateTime_ be the *this* value.
+        1. If Type(_dateTime_) is not Object or _dateTime_ does not have an [[InitializedTemporalDateTime]] internal slot, then
+          1. Throw a *TypeError* exception.
+        1. Return ? CreateDate(_dateTime_.[[Year]], _dateTime_.[[Month]], _dateTime_.[[Day]]).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause>
+      <h1>Temporal.DateTime.prototype.getYearMonth ( )</h1>
+      <p>
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _dateTime_ be the *this* value.
+        1. If Type(_dateTime_) is not Object or _dateTime_ does not have an [[InitializedTemporalDateTime]] internal slot, then
+          1. Throw a *TypeError* exception.
+        1. Return ? CreateYearMonth(_dateTime_.[[Year]], _dateTime_.[[Month]]).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause>
+      <h1>Temporal.DateTime.prototype.getMonthDay ( )</h1>
+      <p>
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _dateTime_ be the *this* value.
+        1. If Type(_dateTime_) is not Object or _dateTime_ does not have an [[InitializedTemporalDateTime]] internal slot, then
+          1. Throw a *TypeError* exception.
+        1. Return ? CreateMonthDay(_dateTime_.[[Month]], _dateTime_.[[Day]]).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause>
+      <h1>Temporal.DateTime.prototype.getTime ( )</h1>
+      <p>
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _dateTime_ be the *this* value.
+        1. If Type(_dateTime_) is not Object or _dateTime_ does not have an [[InitializedTemporalDateTime]] internal slot, then
+          1. Throw a *TypeError* exception.
+        1. Return ? CreateTime(_dateTime_.[[Hour]], _dateTime_.[[Minute]], _dateTime_.[[Second]], _dateTime_.[[Millisecond]], _dateTime_.[[Microsecond]], _dateTime_.[[Nanosecond]]).
+      </emu-alg>
+    </emu-clause>
   </emu-clause>
 
+  <emu-clause id=sec-temporal-datetime-abstract-ops>
+    <h1>Abstract operations</h1>
+
+    <emu-clause id=sec-temporal-ToPartialDateTime aoid=ToPartialDateTime>
+      <h1>ToPartialDateTime ( _dateTime_ )</h1>
+      <emu-alg>
+        1. If Type(_dateTime_) is String, then
+          1. Return ? ParseDateTimeString(_dateTime_).
+        1. If Type(_dateTime_) is Object, then
+          1. Let _result_ be the new Record {
+            [[Year]]: *undefined*,
```

Comment:
```suggestion
              [[Year]]: *undefined*,
```

Reply:
Just the one line?

Reply:
All of them, but GitHub doesn’t yet have multiline suggestions.